lib/geoapify_client.rb

```diff
@@ -0,0 +1,11 @@
+module GeoapifyClient
+    class << self
+        def connection
+            connection ||= Faraday.new(
+                url: 'https://api.geoapify.com/',
+                params: { apiKey: ENV['GEOAPIFY_KEY'] },
```

Comment:
Optional: If you want your environment to fatally error if you haven't configured a key, you can use `fetch` with no second argument.

```suggestion
                params: { apiKey: ENV.fetch('GEOAPIFY_KEY') },
```